pubspec.yaml

```diff
@@ -2,16 +2,16 @@ name: very_good_infinite_list
 description: >-
   A library for easily displaying paginated data, created by Very Good Ventures.
   Great for activity feeds, news feeds, and more.
-version: 0.7.1 
+version: 0.7.1
 repository: https://github.com/VeryGoodOpenSource/very_good_infinite_list
 issue_tracker: https://github.com/VeryGoodOpenSource/very_good_infinite_list/issues
 homepage: https://github.com/VeryGoodOpenSource/very_good_infinite_list
 documentation: https://github.com/VeryGoodOpenSource/very_good_infinite_list
 topics: [flutter, list-view, scroll, widgets]
 
 environment:
-  sdk: ">=2.18.0 <4.0.0"
-  flutter: ">=3.0.0"
+  sdk: ">=3.0.0 <4.0.0"
+  flutter: ">=3.10.0"
```

Comment:
Those using a Dart SDK lower than 3.0.0 would have to upgrade if they want to keep using the latest release.

For us to test the package on the minimum Dart SDK we need to increase this, since currently Very Good Analysis requires Dart 3.0.0 and it is not possible to get dependencies on anything lower than 3.0.0. Hence, we can't guarantee that the package is running as expected for versions lower than 3.0.0 because we can't test it using those versions.

Reply:
Still a breaking change :D

Reply:
Yes, I updated the Pull Request naming accordingly 🙌